packages/scheduler/tests/formatDuration.test.ts

```diff
@@ -0,0 +1,23 @@
+import { hrToSeconds } from "../src/formatDuration";
```

Comment:
Not asking for changes here, just a minor note. I like the tests being in another folder for improving the inner loop times because they get automatically separated out. But one drawback here is that I have to manually jump between /tests/ and /src/ to see what's tested and what isn't. In this case you added a `fixures` folder to stick `formatDuration.test.ts` in, but there's no `fixtures` folder in `src`.  There is some mental jumps to validate what things are tested.

I'm not sure what's better to optimize; inner loop savings time (awesome!) or being able to easily spot what is tested (also useful but maybe it's less important?)

Reply:
Not entirely sure! I just chose this because of my previous attempts with __test__ seemed kind of ugly to me...